panel/layout/fade.py

```diff
@@ -0,0 +1,165 @@
+"""
+The Fade layout enables you to quickly compare multiple panels
+"""
+from __future__ import annotations
+
+from typing import ClassVar, List
+
+import param
+
+from ..io.resources import CDN_DIST
+from ..reactive import ReactiveHTML
+from .base import ListLike
+
+
+class Fade(ListLike, ReactiveHTML):
+    """
+    The Fade layout enables you to quickly compare multiple objects;
+    laid on top of each other with a slider to fade between them.
+    For example, if there are two objects, the first panel will be
+    fully opaque when the slider is at 0%, the second panel will be
+    fully opaque when the slider is at 100%, and both objects will be
+    half opaque when the slider is at 50%.
+    This can also support more than two objects and will gradually
+    fade between object 0 to object 1 to object 2.
+    """
+
+    objects = param.List(
+        default=[],
+        doc="The list of child objects that make up the layout.",
+        precedence=-1,
+    )
+
+    slider_width = param.Integer(
```

Comment:
I don' think that `Fade` should include a slider. I think `Fade` should be a low level interface that can be combined into other elements. For example I might want to have a periodic callback changing the `value`. Or I might want to use the `Player` widget.

Its the same challenge as the `ChatBox`. Do we build the low level components that users can combine? Do we build opinionated, high level interfaces that serves specific use cases? Or do we build both.

So far I believe we've been building the low level components.

I need some guidance from you @philippjfr here on what are our core principles.

Reply:
This was based off [`Swipe`](https://github.com/holoviz/panel/blob/main/panel/layout/swipe.py#L129) but I can understand linking it to player. Maybe as a compromise it can have a `show_slider` param?